src/Config/CS.props

```diff
@@ -66,4 +66,9 @@
     <Prefer32Bit>false</Prefer32Bit>
 	<UICulture>en-US</UICulture>
   </PropertyGroup>
+  <!--This is used to set the framework version for icon resource dll generation (lookup of system.drawing.dll)-->
+  <!--TODO For now this is set to 4.6 to support build machines without vs2017- but should be set to match the project targets-currently 4.7-->
```

Comment:
Where is the place set to 4.6?

Reply:
whoops fixed this, can get rid of this comment now.